pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -214,276 +210,275 @@ let
     fi
   '';
 
-in
-
-assert allPkgconfigDepends != [] -> pkgconfig != null;
-
-stdenv.mkDerivation ({
```

Comment:
Is there any way to avoid re-indenting everything? Makes it harder to review and kills git blame.

Reply:
Maybe we can just do `fix (drv: stdenv.mkDerivation ({` and add a paren at the end?

Reply:
You could add `-w` to `git diff` or `git blame` :)

I guess the github blame's won't work with this though. I could of course just not indent it, but then it's kinda ugly

Reply:
Or `fix`, yeah that sounds better